tools/asciidoctor/cli.adoc

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) 2020  Matthew Glazar
+// See end of file for extended copyright information.
+
+= quick-lint-js(1)
+:doctype: manpage
+:version: 0.2.0
```

Comment:
Update docs/RELEASE.md to mention this version number (and to un-mention quick-lint-js.1?)